### PR TITLE
Widget issues

### LINKED ISSUE #7807: [WIDGETS] When adding a widget, we should go directly to the widget options on the left panel

### PR DESCRIPTION
**Pull request by xavijam**
I feel it fixes one or two tickets but I can't find them, @saleiva rings the bell?

- When a new widget is added, it goes to the form view.
- It shows the source info in the widget item view and in the widget form view.

screen shot 2016-06-25 at 11 18 30

screen shot 2016-06-25 at 11 42 20

CR: @nobuti 
Fixes #7807.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "cartodb-ui",
-  "version": "3.27.34",
+  "version": "3.27.35",
```
Comment (nobuti) [inline package.json]: 36

**Comment by nobuti** [comment]
👍  🚀

**Comment by xavijam** [comment]
💋 , to be merged on monday 😬

**Comment by saleiva** [comment]
OMG!

**Pull request merged**